lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -144,6 +168,15 @@ def start
       end
     end
 
+    def start
+      super
+
+      init_api_client()
```

Comment:
no, the metadata service is just a local (to the VM) http server, it doesn't require any auth.